app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -37,14 +38,14 @@ def create_resource(type, id, data)
 
       raise BadRequestError, "invalid resource_type #{data['resource_type']}" unless resource_type
 
-      unless resource_type.respond_to?(:supports_conversion_host?) && resource_type.supports_conversion_host?
-        raise BadRequestError, "unsupported resource_type #{resource_type}"
-      end
-
       collection_type = resource_type.table_name
 
       resource = resource_search(data['resource_id'], resource_type.to_s, collection_class(collection_type))
 
+      unless resource.respond_to?(:supports_conversion_host?) && resource.supports_conversion_host?
```

Comment:
The `.respond_to?` check shouldn't be needed as long as the model includes `SupportsFeatureMixin` since every possible feature is defined as unsupported by default.

Reply:
Ok, updated.